03-Bash/find_dup_files/find_dup_files.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+
+function print_help()
+{
+    echo "Usage: find_dup_files [OPTION] [PATH]"
+    echo "Searches for files with MD5 checksum (128-bit) collisions in the"
+    echo "specified with PATH directory (by default in current directory)."
+    echo ""
+    echo "Option:"
+    echo -e "\t-h, --help\tprint this message and exit."
+}
+
+function find_dup_files()
+{
+    SEARCH_PATH=$1
+    PREV_FILE_PATH=""
+    PREV_CHECKSUM=""
+
+    if [ "${SEARCH_PATH:0:1}" != "/" ]; then
+        cd $SEARCH_PATH
+        SEARCH_PATH=$(pwd)
+        cd - &>/dev/null
+    fi
```

Comment:
What is the reason of ignoring non-absolute paths as target folder? Are these lines required for this task?

Reply:
When I work with such utilities, I prefer to see the absolute path. This is the only reason for usage of these lines. Moreover, since this is not specified in the task, the choice is left to the author.